### ISSUE TITLE
a11y P3 - No line break or space between published and updated time with No CSS enabled

### ISSUE THREAD

**Issue by HarveyPeachey** [bbc/simorgh#5518]
**Describe the bug**
No line break or space between published and updated time - you should be able to tell when the published time ends and the updated time starts.

~~Possibly related https://github.com/bbc/simorgh/issues/5515.~~

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to any page that has timestamp and updated timestamp. https://www.test.bbc.com/pidgin/media-43768328?renderer_env=live
2. Disable CSS.

**Expected behaviour**
There should be a character space between the two timestamps.

**Screenshots**
![Screenshot 2020-02-12 at 11 20 31](https://user-images.githubusercontent.com/4706195/74330952-8e830c00-4d8a-11ea-891e-f98ede65acd9.png)

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
